### PR TITLE
Update developer guide formatting (Dev Ops and User Stories)

### PR DESCRIPTION
**Pull request by hzxcaryn**

### CODE REVIEW

**Comment by shanwpf** [comment]
Any reason why _dependencies_ is in italics in section 7.6?

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -854,17 +849,19 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
-|`*` |employer |view an employee’s photo|recognize the employee.
+|`*` |employer |view an employee’s photo |recognize the employee.
 
-|`*` |employer |remove an employee’s photo|remove their old photo.
+|`*` |employer |remove an employee’s photo |remove their old photo.
 
 |`*` |employee |add a photo |be easily identified by my employer.
 
-|`*` |employee |have notification for the selected slot |be assured that I am working at that slot.
+|`*` |employee |have notification for slot confirmation |be assured that I am working at that slot.
```
Comment (shanwpf) [inline docs/DeveloperGuide.adoc]: `have a notification`

Comment (shanwpf) [inline docs/DeveloperGuide.adoc]: `have a notification`

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -808,44 +810,37 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
 |`* * *` |employee |view the schedule |choose when I want to work.
 
-|`* * *` |employee |select from the available slots |work during that slot.
+|`* * *` |employee |select from the available slots in the schedule |work during that slot.
 
-|`* * *` |employer |add outlet |update the app with new outlets
+|`* *` |employer |edit an employee’s contact |update their outdated contact information.
 
-|`* *` |employer |sort employees by name |have a clearer view of all employees’ details.
+|`* *` |employer |edit outlet information |update outdated outlet information.
```
Comment (shanwpf) [inline docs/DeveloperGuide.adoc]: missing `the` before outlet and outdated

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -503,9 +504,10 @@ Here are the steps to create a new release.
 
 === Managing Dependencies
 
-A project often depends on third-party libraries. For example, Address Book depends on the http://wiki.fasterxml.com/JacksonHome[Jackson library] for XML parsing. Managing these _dependencies_ can be automated using Gradle. For example, Gradle can download the dependencies automatically, which is better than these alternatives. +
-a. Include those libraries in the repo (this bloats the repo size) +
-b. Require developers to download those libraries manually (this creates extra work for developers)
+A project often depends on third-party libraries. For example, PTMan depends on the http://wiki.fasterxml.com/JacksonHome[Jackson library] for XML parsing. Managing these _dependencies_ can be automated using Gradle. One advantage of Gradle is that it can download these _dependencies_ automatically, which is better than the below alternatives: +
```
Comment (shanwpf) [inline docs/DeveloperGuide.adoc]: Any reason why _dependencies_ are in italics?

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -476,6 +476,7 @@ e.g. `seedu.address.logic.LogicManagerTest`
 * Solution: Execute Gradle task `processResources`.
 
 == Dev Ops
+You need to ensure the next few sections are fulfilled so that the project is fully integrated, tested and built for release. You have to be fully aware of the configurations and the infrastructure of the application and the environment of where the application will be installed.
```
Comment (shanwpf) [inline docs/DeveloperGuide.adoc]: `The following sections help you ensure that the project is fully integrated, tested, and built for release.` sounds better. 2nd sentence doesn't seem necessary.

Comment (hzxcaryn) [reply docs/DeveloperGuide.adoc]: Ok thanks!

**Comment by shanwpf** [comment]
What have you done...

**Pull request merged**